libs/dh/metering-point/feature-overview/src/test-setup.ts

```diff
@@ -22,6 +22,8 @@ import {
 } from '@energinet-datahub/gf/test-util-staging';
 import { addDomMatchers } from '@energinet-datahub/gf/test-util-matchers';
 
+import '@energinet-datahub/dh/shared/test-util-msw';
```

Comment:
It's only necessary for tests actually doing HTTP request

Reply:
What about the tests in `metering-point-api-v1.spec.ts`? They are skipped now but the make HTTP requests. Should we import `test-util-msw` in that lib's `test-setup` file?

Reply:
I think the `metering-point-api-v1.spec.ts` should be removed, since it's actually testing the BFF itself (we must expect that the backend is testing their own code). We could create some "health / smoke" tests instead actually calling an environment however this requires that we have a metering point id we can be sure to exist which we don't have at this time.

Reply:
If we're going the MSW way, which we are, then I think it makes sense to remove the file and instead look at other ways to test the connection with the BFF (like you propose).